.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
is it desirable to have the action to run with pushes too? given it's a CI task, shouldn't it run only upon PR creation?

Reply:
Having it run on pushes makes certain things easier - work on forks for example (before the code is ready for PRs to the main repo).  There are a number of difference ways to configure this.  Given the simplicity of this configuration and the minimal resources allocated, I think this is fine.  But if you want something else, that's fine too.